### ISSUE TITLE
[ UX ] - Paging and UIPageControl

### ISSUE THREAD

**Issue by Dekablade01** [pauloec/Nimbl3#8]
From my understanding, when there's `UIPageControl`, `UICollectionView` or `UITableView` 
 `pagingEnable` should be `YES`

**Comment by pauloec** [comment]
Yes, i didn't enable it because images were inconsistent in size and so `pagingEnabled` would look somewhat lousy.

**Comment by Dekablade01** [comment]
You can fix it, by set cell size equal to `viewController.size` and set tableView inset up to navigationBar height.  and enable the `pagingEnable`
